en/faq/deployment-azure-portal.md

```diff
@@ -3,6 +3,51 @@ title: How to deploy on Azure Portal?
 description: How to deploy a Nuxt.js application on Azure Portal?
 ---
 
+## Requirements
+- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site wont start up.
```

Comment:
Is this really the case?s 🤔

Reply:
Yes, from my testing and experience. I was unable to fire up the default nuxt server on on my Azure Web Service.